FrameworkInternals/generateHonkyTonk.py

```diff
@@ -1,30 +0,0 @@
-#!/usr/bin/env python
-# encoding: utf-8
-'''
-generateHonkyTonk.py
-
-@author:     Damian Abalo Miron <damian.abalo@cern.ch>
-@author:     Piotr Nikiel <piotr@nikiel.info>
-
-@copyright:  2015 CERN
-
-@license:
-Copyright (c) 2015, CERN, Universidad de Oviedo.
-All rights reserved.
-Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
-1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
-2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
-@contact:    quasar-developers@cern.ch
-'''
-
-import os
-from transformDesign import transformDesignVerbose
-
-def generateHonkyTonk():
-	"""Generates honkyTonky.cc, a special class intended for testing your namespace with simulated (random) values."""

```

Comment:
have we lost this documentation as to what the command
quasar.py generate honkyTonk MY_CLASS
actually does?

I feel end users probably genuinely would profit from having some help as to what this command does.

Reply:
@ben-farnham,
my proposal to solve it is the following: when an user calls any command with -h, a web browser would be opened, positioned on an anchor (a paragraph) about particular command.

This can be done using Python's standard module:
import webbrowser
webbrowser.open('http://google.com')

advantages:
* would let reuse the same clean python logic,
* the documentation could be of better quality (one can embed figures, tables in HTML instead of few lines of text in python's doc) and it could be more detailed

I could make an initial quasarCommands.html copy-pasting the existing documentation and aim for refinement as we go. Makes sense?

Reply:
Yep, makes sense.

Reply:
Thanks. It will be done under another ticket:
https://its.cern.ch/jira/browse/OPCUA-1111
aimed to enter the same quasar release.
I will merge this PR today when the final CI check is finished.